tests/rustdoc-ui/doc-attribute-unsupported.rs

```diff
@@ -0,0 +1,5 @@
+// This is currently not supported but should be!
```

Comment:
What happens tho if you add `#![feature(rustdoc_internals)]`?

Reply:
Arg, I tested with a file with all attributes and forgot to copy/paste this one. ><